docs/setup-react-native.md

```diff
@@ -0,0 +1,49 @@
+# Setup React Native
+
+## Prerequisites
+
+To start using Aries Framework JavaScript in React Native some platform specific dependencies are required.
+
+1. Follow the [React Native Setup](https://reactnative.dev/docs/environment-setup) guide to set up your environment.
+2. Add `rn-indy-sdk` and `aries-framework` to your project.
+
+```bash
+yarn add aries-framework rn-indy-sdk    # npm install aries-framework rn-indy-sdk
+```
+
+3. Install [Libindy](https://github.com/hyperledger/indy-sdk) for iOS and Android:
+
+   - [iOS](../docs/libindy/ios.md)
+   - [Android](../docs/libindy/android.md)
+
+4. If you're using React Native > 0.61.5, make sure you have Hermes enabled, as the app will crash on Android when opening a ledger pool. See the React Native [docs](https://reactnative.dev/docs/hermes) on Hermes on how to enable Hermes.
+   - Indy SDK [issue](https://github.com/hyperledger/indy-sdk/issues/2346#issuecomment-841000640)
+
+## Agent Setup
+
+Initializing the Agent also requires some React Native specific setup, mainly for the Indy SDK and File System. Below is a sample config, see the [docs](./README.md) for an overview of getting started guides. If you want to jump right in, check the [Getting Started: Agent](./getting-started/0-agent.md) guide.
+
+```ts
+import { Agent } from 'aries-framework'
+
+// Import rn-indy-sdk and File System for React Native
+import indy from 'rn-indy-sdk'
+import { ReactNativeFileSystem } from 'aries-framework/build/src/storage/fs/ReactNativeFileSystem'
+
+// This creates an agent with all the specified configuration data
+const agent = new Agent({
+  label: 'my-agent',
+  walletConfig: { id: 'walletId' },
+  walletCredentials: { key: 'testkey0000000000000000000000000' },
+  indy,
+  fileSystem: new ReactNativeFileSystem(),
+})
+
+// Make sure to initialize the agent before using it.
+try {
+  await agent.init()
+  console.log('Initialized agent!')
+} catch (error) {
+  console.log(error)
+}
+```
```

Comment:
I kind of miss the update I did in this commit ec95d83d  about TypeScript and known errors here.